README.md

```
@@ -76,25 +76,34 @@ The interviews feature is protected by a feature flag. By default, interviews pa
 #### How to enable/disable interviews:
 
 - **Local development:**
-  - Add to your `.env` file:
+  - Add to your `.env.local` file:
     ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)